assignments/week-0.md

```diff
@@ -3,26 +3,32 @@
 ## Assignment Title: hello-world
 
 ## Instructions:
+
 Before you start work on your assignment complete the following steps to get GitHub setup correctly. This is how you will share your work with your mentor and Code Apprentice.
 
-1. Create a new repo in github titled `hello-world`
-1. Copy the url titled `quick setup` it should start with https:// This will be different for everyone and should should replace `{origin-from-your-github-repo}` with that link.
-1. Open a terminal
-1. Copy and paste the following commands into your terminal.
+1. Create a new repo in GitHub titled `hello-world`
+2. Copy the url titled `quick setup` it should start with https:// This will be different for everyone and should should replace `{origin-from-your-github-repo}` with that link.
+3. Open a terminal
+4. Copy and paste the following commands into your terminal.
 	- `cd ~ && mkdir workspace`
 	- `cd workspace && mkdir hello-world`
+	- `cd hello-world`
 	- `git init`
 	- `git remote add origin {origin-from-your-github-repo}`
 	- `git add .`
 	- `git commit -m 'init commit'`
 	- `git push origin master`
+
+	(For reference, `cd` stands for `change directory` and `mkdir` stands for `make directory`. Just after creating the needed workspace and project directories, you are initializing git in that project and connecting it to the specific GitHub repository. The last steps of `add`, `commit`, and `push` are commands that you'll use often to update your changes with GitHub. Research any of these if they don't make sense!)
   
-1. Create a github repo, and include a markdown file with your name. This is a test just to make sure that you understand the importance of github. This will also allow the Code Apprentice team to ensure that your account is entered correctly into our system.
+5. Create a GitHub repo, and include a markdown file with your name. This is a test just to make sure that you understand the importance of GitHub. This will also allow the Code Apprentice team to ensure that your account is entered correctly into our system.
+
+---
 
-Once you have completed the code do the following
+Once you have completed the code do the following:
 
 1. Open a terminal
-1. `cd ~/workspace/hello-world`
-1. `git add .`
-1. `git commit -m 'finished the assignment'`
-1. `git push origin master`
```

Comment:
this was actually intentional - .md files will enumerate correctly for ordered lists as long as the integer is repeated. Kind of goofy but it does help with longer documents. Of course this is just you know my opinion haha

Reply:
@garrett-vangilder I have some keybindings that resolve this for me, but I get what you mean!